src/app/pages/OptionsPage/OptionsPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './OptionsPage.module.css';
+import Header from '../../components/Header/Header';
+import Nav from '../../components/Nav/Nav';
+import LogInSignInLink from '../../components/LogIn-SignIn-Link/LogIn-SignIn-Link';
+
+function OptionsPage(): JSX.Element {
+  return (
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <div className={styles.linkWrapper}>
+          <LogInSignInLink url="/signIn" value="Sign in" />
+          <LogInSignInLink url="/LogIn" value="Log in" />
```

Comment:
Please use consistes casing.
```suggestion
          <LogInSignInLink url="/signin" value="Sign in" />
          <LogInSignInLink url="/login" value="Log in" />
```